src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Shuffle.cs

```diff
@@ -12,15 +12,13 @@
 namespace Microsoft.PowerFx.Core.Texl.Builtins
 {
     // Shuffle(source:*)
-    internal sealed class ShuffleFunction : BuiltinFunction
+    internal sealed class ShuffleFunction : FunctionWithTableInput
     {
         // Multiple invocations with the same args may result in different results.
         public override bool IsStateless => false;
 
         public override bool IsSelfContained => true;
 
-        public override bool SupportsParamCoercion => false;
```

Comment:
>SupportsParamCoercion => false [](http://example.com/codeflow?start=29&length=30)

The (new) base class (FunctionWithTableInput) already forbids coercions for the first argument (and this function only has one)